src/server/ideController.py

```diff
@@ -371,17 +384,23 @@ def _save_callback(self, filename, changes, version, users):
       }
     """
 
-    temp_message = ("Temp msg from controller until "
-                    "c++ module will return applied modifications")
-    changes = [dict(type=self.CHANGE_ADD_TYPE, pos=0, content=temp_message)]
+    # Dirty hack to know type
```

Comment:
@grondinjc Do you have another idea to remove this hack later? I feel like we should take care of this now or really soon.